bower.json

```diff
@@ -36,6 +36,11 @@
 		"jquery-1.11.1": "jquery#1.11.1",
 		"jquery-1.11.2": "jquery#1.11.2",
 		"jquery-1.11.3": "jquery#1.11.3",
+		"jquery-1.12.0": "jquery#1.12.0",
+		"jquery-1.12.1": "jquery#1.12.1",
+		"jquery-1.12.2": "jquery#1.12.2",
+		"jquery-1.12.3": "jquery#1.12.3",
+		"jquery-1.12.4": "jquery#1.12.4",
 		"jquery-1.x": "jquery#1.11.3",
```

Comment:
Changed. I also changed it at https://github.com/jquery/jquery-ui/blob/fbc79e1bda2b58f980b9c6fedf3f8ddbce294d7b/tests/lib/bootstrap.js#L99